File: examples/java_toy/Calc.java
package toy;

/** Instance methods for Class.method qnames. */
public class Calc {
  public int scale(int v) {
    return MathUtil.add(v, 0);
  }

  public int run(int a, int b) {
    return scale(MathUtil.compute(a, b));
  }
}

File: examples/csharp_toy/Calc.cs
namespace Toy;

/** Instance methods for Class.method qnames. */
public class Calc
{
    public int Scale(int v)
    {
        return MathUtil.Add(v, 0);
    }

    public int Run(int a, int b)
    {
        return Scale(MathUtil.Compute(a, b));
    }
}
